underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java

```diff
@@ -115,29 +117,18 @@ public void connectFromWorker(Configuration conf, String hostname) {
 
   @Override
   public OutputStream create(String path) throws IOException {
+    return create(path, new UnderFileSystemCreateOptions(mConfiguration));
+  }
+
+  @Override
+  public OutputStream create(String path, UnderFileSystemCreateOptions options) throws IOException {
+    // Block size and permission status in options are ignored when creating GCS object.
```

Comment:
This comment will get stale when new fields are added to `options`. I would simply remove it.